### ISSUE TITLE
if the browser plugin ScriptSafe plugin is used, the javascript disabled message is not shown

### ISSUE THREAD

**Issue by fpietrosanti** [globaleaks/globaleaks-whistleblowing-software#925]
If Javascript is disabled, the end-user is not shown proper error messages.

In past we made a proper error message to appear, informing the user about the need to enable javascript. 

Now sounds like is not working anymore, the user is not informed that he must enable javascript.

**Comment by vecna** [comment]
what we can do about that ?

**Comment by hellais** [comment]
I think my above suggestion is still a viable option that is quite simple to implement and does not require external dependencies.

**Comment by evilaliv3** [comment]
you are right @hellais.

while implementing this we should keep in mind problems like: https://github.com/globaleaks/GlobaLeaks/issues/1189

**Comment by hellais** [comment]
@evilaliv3 I don't think we should be investing time in supporting a browser that will soon be deprecated (OrWeb will be replaced by OrFox).

**Comment by evilaliv3** [comment]
i see and i agree. what we can do is in general to keep in mind this and find general solutions and design well written messagise and explainations of what is happening and how to point the user to solve his problem.

**Comment by fpietrosanti** [comment]
@evilaliv3 Orbot is widely diffused and it's unlikely to go away in few months (maybe in 2 years?), btw sounds like a similar problem to provide better instructions for individual widely used browsers on how to enable JS.

**Comment by evilaliv3** [comment]
Closing as completed long time ago.
